src/components/ReviewsPage/ReviewsPage.tsx

```diff
@@ -1,7 +1,262 @@
+    });
+    setReviewToEdit(null);
+  };
+
+  if (loading) return <CircularProgress color="secondary" />;
+  if (error) return <Alert severity="error">Fetching data failed</Alert>;
+
   return (
-    <div>
-      <h1>ReviewsPage</h1>
-    </div>
+    <TableContainer component={Paper}>
+      <Box display="flex" justifyContent="space-around">
+        <Typography component="h1" variant="h2">
+          Restaurant Reviews
+        </Typography>
+        <Button
+          variant="contained"
+          color="secondary"
+          onClick={() => setIsOpenCreateReview(true)}
+        >
+          Add review
+        </Button>
+      </Box>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Author ID</TableCell>
+            <TableCell>Author First Name</TableCell>
+            <TableCell>Author Last Name</TableCell>
+            <TableCell>Rating</TableCell>
+            <TableCell>Feedback</TableCell>
+            <TableCell>Created At</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {data?.findAllRestaurantReviews.edges &&
+            data.findAllRestaurantReviews.edges.map(({ node }) => (
+              <TableRow key={node.id}>
+                <TableCell>{node.author.id}</TableCell>
+                <TableCell>{node.author.firstName}</TableCell>
+                <TableCell>{node.author.lastName}</TableCell>
+                <TableCell>{node.rating}</TableCell>
+                <TableCell>{node.feedback}</TableCell>
+                <TableCell>
+                  {new Date(parseInt(node.createdAt)).toLocaleString()}
+                </TableCell>
+                <TableCell>
+                  <Button
+                    variant="contained"
+                    color="secondary"
+                    onClick={() => setReviewToEdit(node)}
+                  >
+                    Edit review
+                  </Button>
+                </TableCell>
+                <TableCell>
+                  <Button
+                    variant="contained"
+                    color="secondary"
+                    onClick={() =>
+                      handleDeleteReview({ id: node.id.toString() })
+                    }
+                  >
+                    Delete review
+                  </Button>
+                </TableCell>
+              </TableRow>
+            ))}
+        </TableBody>
+      </Table>
+      {isOpenCreateReview && (
+        <Dialog
+          open={isOpenCreateReview}
+          onClose={() => setIsOpenCreateReview(false)}
+          component="fieldset"
+        >
+          <Box sx={{ padding: "30px 20px" }}>
+            <DialogTitle component="legend" variant="h4" sx={{ p: 0 }}>
+              Please create a review
+            </DialogTitle>
+            <ReviewForm
+              setIsOpen={setIsOpenCreateReview}
+              onSubmit={handleCreateReview}
+            />
+          </Box>
+        </Dialog>
+      )}
+      {reviewToEdit && (
```

Comment:
Remove `reviewToEdit &&`, the same for create modal. Visibility of modals should depend on `open` prop, not conditional rendering